core/ui/mpc/keysign/KeysignSigningStep.tsx

```
@@ -37,123 +46,218 @@ export const KeysignSigningStep = ({ onBack }: KeysignSigningStepProps) => {
+          const txResult = getLastItem(txResults)
+          const handleFinish = () =>
+            isDAppSigning ? window.close() : navigate({ id: 'vault' })
+          return (
+            <>
+              <PageHeader title={t('overview')} hasBorder />
+              <MatchRecordUnion
+                value={payload}
+                handlers={{
+                  keysign: payload => {
+                    const { swapPayload } = payload
+                    const isSwapTx = swapPayload && swapPayload.value
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)